helm/ffc-elm-scheme-service/Chart.yaml

```diff
@@ -1,6 +1,11 @@
-apiVersion: v1
+apiVersion: v2
 appVersion: "1.0"
 description: A Helm chart for Kubernetes
 name: ffc-elm-scheme-service
 # version is overwritten by the build pipline
 version: 1.0.0
+
+dependencies:
+  - name: ffc-helm-library
+    version: ^1.0.0
```

Comment:
Might be worth starting with the latest version which at present is 1.1.2 rather than 1.0.0.  

There is some config you have that wouldn't work with 1.0.0.  Just in case the public charts move from GitHub (as they may well), you know that your build will fail early if the minimum you need isn't in the new location.

Reply:
Thanks, good catch!